google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -47,10 +47,52 @@ class Bucket
         attr_accessor :gapi
 
         ##
+        # A `true`/`false` value or a project ID string to be sent as the
```

Comment:
I misunderstood its context when displayed in documentations. One last nit but not required. 

Why `true`/`false` and not boolean?